srcpkgs/ncspot/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'ncspot'
+pkgname=ncspot
+version=0.2.0
+revision=1
+build_style="cargo"
+makedepends="ncurses-devel python3 dbus-devel xcb-util-renderutil-devel libxcb-devel libressl-devel pulseaudio-devel pkg-config"
```

Comment:
`pkg-config` should be a `hostmakedepends`

Try to keep your lines to a limit of 80 characters too.